meson.build

```diff
@@ -520,6 +520,8 @@ sdb_exe = executable('sdb', ['shlr/sdb/src/main.c'] + sdb_files,
   ],
   implicit_include_directories: false,
   native: true,
+  install: true,
+  rename: 'r2sdb'
```

Comment:
@zamaudio 
```
../meson.build:517: WARNING: Passed invalid keyword argument "rename".
WARNING: This will become a hard error in the future.
WARNING: Installing target build for the build machine. This will fail in a cross build.
```

Reply:
It seems like `rename` is not ok and also it is not ok to install this executable because this is compiled for the build machine, not the target machine (which may be different if you cross compile for android, for example). Could you fix this? I'm not sure what to do, but r2sdb should not be installed. Maybe radare2-extras should just require sdb installed, but it should get it directly from sdb repository.

Reply:
Hmm, that's weird.  Is there a repo for sdb I can use for CI testing radare2-extras that require sdb?

Reply:
What do you mean? https://github.com/radareorg/sdb ?

Reply:
OK thanks, I just submitted a partial revert for the r2sdb stuff.